Article/Article.js

```diff
@@ -1,29 +1,104 @@
-// Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+
+/*
+To add the new HTML content, had to add the new added code on top here
+before the class Article so the newDiv can inherit
+all the proper CSS styling (especially for the expand button).
+ */
+
+const selectOne = tag => document.querySelector(tag);
+const selectAll = tag => document.querySelectorAll(tag);
+const createEle = tag => document.createElement(tag);
+
+const articleContainer = selectOne('.articles');
+
+const createElementAndAppend = ({tag, attributes = {}, content, targetElement, styles = {}}) => {
```

Comment:
This is really cool. You were allowed to edit the html on this one, but staying fresh with last week's stuff is never a bad idea.